calibrate.py

```diff
@@ -802,26 +861,30 @@ def capture_images_sync(self):
                 else:
                     self.show_failed_capture_frame()
                     capturing = False
-
             # print(f'self.images_captured_polygon  {self.images_captured_polygon}')
             # print(f'self.current_polygon  {self.current_polygon}')
             # print(f'len(self.polygons)  {len(self.polygons)}')
 
             if self.images_captured_polygon == self.args.count:
                 self.images_captured_polygon = 0
                 self.current_polygon += 1
+                if self.args.numPictures == None:
+                    if self.current_polygon == len(self.polygons):
+                        finished = True
+                        cv2.destroyAllWindows()
+                        break
+                else:
+                    if self.images_captured == self.args.numPictures:
+                        finished = True
+                        cv2.destroyAllWindows()
+                        break
 
-                if self.current_polygon == len(self.polygons):
-                    finished = True
-                    cv2.destroyAllWindows()
-                    break
 
 
 
     def calibrate(self):
         print("Starting image processing")
-        stereo_calib = calibUtils.StereoCalibration()
-        stereo_calib.traceLevel = self.args.traceLevel
+        stereo_calib = calibUtils.StereoCalibration(self.args.traceLevel, self.args.outputScaleFactor, self.args.disableCamera)
```

Comment:
This arguments are deployed in https://github.com/luxonis/depthai-calibration/tree/added_disableCamera not in main branch of depthai_calibration.
@saching13. Should we use then this branch for now as a calibration submodule?
And when changes are deployed in https://github.com/luxonis/Factory-calibration-DepthAI push them to main?

Reply:
If the same feature is coved here. and no other features are coming from your branch. yes. we can use this branch

Reply:
Some features are not set properly in this PR (traceLevel, outputScaleFactor, disableRGB, lensPosition) so I created new PR from main and used changes @CenekAlbl suggested.
https://github.com/luxonis/depthai/pull/1099